apps/pay/app/[username]/page.tsx

```diff
@@ -28,11 +31,18 @@ gql`
   }
 `
 
-function ReceivePayment() {
-  const router = useRouter()
-  const { username, memo, display } = router.query
-  const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent)
-  const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
+type ReceivePaymentProps = {
```

Comment:
I think generally we can just call this `Props`, assuming there is typicall always just one per file, it's not needed to narrow down the variable name. 

```suggestion
type Props = {
```